Common info
URPoseSPToUniRicochet ricochet
bool moving
string actual_pose
string moveit_state
string moveit_message
